### ISSUE TITLE
Incorrect line reported for signature broken over multiple lines with comments

### ISSUE THREAD

**Issue by dalemyers** [peripheryapp/periphery#243]
The easiest way to show this is an example. Here's a minimal repro:

```swift
import UIKit

public protocol SomeProtocol {
    func doThing(_ var1: Int,
                  // Some comment here
                  for var2: UIViewController)
}

class Whatever: SomeProtocol {
    func doThing(_ var1: Int, for var2: UIViewController) {
        print("hello \(var1)")
    }
}

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        (Whatever() as SomeProtocol).doThing(1, for: self)
    }
}
```

If you run periphery, you'll get:
```
/Users/dalemy/Desktop/PeripheryTest/PeripheryTest/ViewController.swift:5:1: warning: Parameter 'var2' is unused
```

However, line 5 is the comment, line 6 is where the issue actually is. If you remove the comment, the line is reported correctly.

**Comment by ileitch** [comment]
Thanks for reporting, this is now fixed in 2.3.3.

**Comment by dalemyers** [comment]
You know, with fixing these issues so quickly, you are ruining me for other projects. 😆 

Thanks!
